test/tests.transform.js

```diff
@@ -47,3 +47,102 @@ describe("_blockHoist nodes", function() {
     assert.deepEqual(names, ["hoistMe", "doNotHoistMe", "oyez"]);
   });
 });
+
+context("functions", function() {
```

Comment:
@modosc as maintainer of the redux-saga im a little bit surprised that this make u a huge difference. I thought this is more 'nice to have' and 'always better have this in place' rather than a game changer. Do u have any numbers for ur project? What will be tree shaken after this change?

I see when this might make a change when u have some shared saga utils module used between projects, but i think most dont